defaults/events/MergeGroup.dhall

```diff
@@ -0,0 +1 @@
+{ types = None (List Text) }
```

Comment:
Hi @michael-wisely-gravwell Thanks so much for this. I wounder if we should represent the activity types as a union type here 🤔 like an "enum".

Reply:
I like that idea! I've pushed a commit that uses a union. The implementation is similar to that of `Release`.